app/models/course_enrichment.rb

```diff
@@ -21,7 +21,7 @@
 
 class CourseEnrichment < ApplicationRecord
   include TouchCourse
-  enum status: %i[draft published rolled_over withdrawn]
```

Comment:
Does this have any impact on the data at all? I'd rather skip linting on these rules to avoid any risk